RecoEgamma/EgammaIsolationAlgos/python/electronTrackIsolations_cfi.py

```diff
@@ -0,0 +1,92 @@
+import FWCore.ParameterSet.Config as cms
+
+trkIsol03CfgV1= cms.PSet(
+    barrelCuts=cms.PSet(
+        minPt=cms.double(0.7),
+        maxDR=cms.double(0.3),
+        minDR=cms.double(0.00),
+        minDEta=cms.double(0.015),
+        maxDZ=cms.double(0.2),
+        maxDPtPt=cms.double(-1),
+        minHits=cms.int32(-1),
+        minPixelHits=cms.int32(-1),
+        allowedQualities=cms.vstring(),
+        algosToReject=cms.vstring("jetCoreRegionalStep")
+        ),
+    endcapCuts=cms.PSet(
```

Comment:
endcap cuts are identical to barrel cuts.
Please define a default and initialize/copy from it to reduce copy-paste

Reply:
done